distributedcombigrid/tests/test_manager.cpp

```diff
@@ -1,3 +1,4 @@
+#ifdef RUN_BOOST_TESTS
```

Comment:
Why did you include this compiler switch here? Shouldn't test manager be completely outsourced into the test_ftolerance.cpp?

Reply:
I had disabled test_manager using this switch instead of deleting it while merging those two files. Now that test_ftolerance works, I can delete it.